packages/apps/transfer/src/services/faucet/index.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ChainwebChainId,
+  ChainwebNetworkId,
+  IPollResponse,
+} from '@kadena/chainweb-node-client';
+import { ICommandBuilder, IPactCommand, Pact } from '@kadena/client';
+import { genKeyPair, sign } from '@kadena/cryptography-utils';
+import { PactNumber } from '@kadena/pactjs';
+
+import { generateApiHost } from '../utils/utils';
+
+import { kadenaConstants } from '@/constants/kadena';
+import { env } from '@/utils/env';
+
+const NETWORK_ID: ChainwebNetworkId = 'testnet04';
```

Comment:
this can be gotten from the constants file

Reply:
Done